### PR TITLE
pjscrape 0.1.4 (new formula)

### PR DESCRIPTION
**Pull request by milaney**

### CODE REVIEW

File: Library/Formula/pjscrape.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require "formula"
```
Comment (DomT4) [inline Library/Formula/pjscrape.rb]: :scissors: this line.

File: Library/Formula/pjscrape.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  head "https://github.com/nrabinowitz/pjscrape.git"
+
+  depends_on "phantomjs"
+
+  def install
+    libexec.install Dir["*"]
+
+    sh = bin/"pjscrape"
+    sh.write <<-EOS.undent
```
Comment (DomT4) [inline Library/Formula/pjscrape.rb]: Can probably just do ` (bin/"pjscrape").write <<-EOS.undent ` here, since the binary name isn't too long.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by mikemcquaid** [comment]
Checksum seems wrong: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26862/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_pjscrape/

Also, please fix the audit warnings: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26862/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_pjscrape___strict/

**Comment by DomT4** [comment]
Any progress on this?

**Comment by DomT4** [comment]
Ping.

**Comment by DomT4** [comment]
Presuming dead from the lack of response to 3 maintainer notifications in ~21 days. Closing here.

If still interested please fix-up the various issues highlighted and resubmit. Thanks for the PR!

**Comment by milaney** [comment]
Apologies; very fair. Work deadlines have squeezed my open source time to
zero, but thanks for the review. Coincidentally, I was rather unenamoured
with the package in question anyway, so I feel Homebrew's coverage of the
more powerful/comprehensive base library for pjscrape is a better use of
maintainer's time.

Kind regards.

MCM/mm
Cheers,
*Max Milaney*, FRSA MACS CP MAICD MBCS MCITP
96 Brickport Road, Park Grove TAS 7320
+61 (0)458 913 634
This message contains confidential information and is intended only for the
individual named. It may also be privileged or otherwise protected by work
product immunity or other legal rules. If you are not the named addressee
you should not disseminate, distribute or copy this e-mail. Please notify
the sender immediately by e-mail if you have received this e-mail by
mistake and delete this e-mail from your system. E-mail transmission cannot
be guaranteed to be secure or error-free as information could be
intercepted, corrupted, lost, destroyed, arrive late or incomplete, or
contain viruses. The sender therefore does not accept liability for any
errors or omissions in the contents of this message, which arise as a
result of e-mail transmission. If verification is required please request a
hard-copy version. Maximillian Milaney, 96 Brickport Road, Park Grove,
Tasmania, Australia 7320.

**Pull request closed without merging**